spinnaker-jenkins-to-vmss/setup-scripts/config_azure_jenkins_storage.sh

```diff
@@ -0,0 +1,446 @@
+#!/bin/bash
```

Comment:
One of the Jenkins jobs that are pre-provisioned on the image use it, I do not want to break that.